bin/navi

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
`<repo>/navi` is the entrypoint of the tool, so we can't move it to `<repo>/bin/navi`: it'll break people's symlinks and the homebrew formula. Please keep it at `<repo>/navi`